### PR TITLE
aws-robomaker-small-warehouse-world: 1.0.1-1 in 'foxy/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by cheng-kevin**
Increasing version of package(s) in repository `aws-robomaker-small-warehouse-world` to `1.0.1-1`:

- upstream repository: https://github.com/aws-robotics/aws-robomaker-small-warehouse-world.git
- release repository: https://github.com/aws-gbp/aws_robomaker_small_warehouse_world-release.git
- distro file: `foxy/distribution.yaml`
- bloom version: `0.10.7`
- previous version for package: `null`

### CODE REVIEW

File: foxy/distribution.yaml [hunk 1]
```diff
@@ -341,6 +341,19 @@ repositories:
...
+      type: git
+      url: https://github.com/aws-robotics/aws-robomaker-small-warehouse-world.git
+      version: ros2
+    release:
+      packages:
+      - aws_robomaker_small_warehouse_world
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/aws-gbp/aws_robomaker_small_warehouse_world-release.git
+      version: 1.0.1-1
```
Comment (hidmic) [inline foxy/distribution.yaml:355]: @cheng-kevin no `source` entry?

**Review by hidmic — COMMENTED** [review]
[Holding for Foxy sync](https://discourse.ros.org/t/preparing-for-foxy-sync-and-patch-release-2021-09-02/22085).

File: foxy/distribution.yaml (hunk 1 above)
Comment (cheng-kevin) [reply foxy/distribution.yaml:355]: added!

**Review by hidmic — APPROVED** [review]

**Comment by hidmic** [comment]
@cheng-kevin would you mind merging master into your branch? That should fix CI.

**Pull request merged**